src/libs/Url.ts

```diff
@@ -11,6 +11,13 @@ function addTrailingForwardSlash(url: string): string {
     return url;
 }
 
+function addLeadingForwardSlash(url: string): string {
```

Comment:
What is the purpose of this method as opposed to just updating `TRIP_ID_PATH` in CONST? Said differently, what re-usability do you see from this method?

Reply:
@stitesExpensify this is to **guarantee** that a path with or without leading slash would work.
If we don't do this, someone adding a new path without a leading slash would be really confused why redirect isn't working. There aren't any errors for it either.